api/src/s2-api.ts

```diff
@@ -120,11 +120,7 @@ export function getPaperUncached(s2Id: string, apiKey?: string): AxiosPromise<Pa
   if (apiKey) {
     conf.headers['x-api-key'] = apiKey;
   }
-  const apiOrigin = (
-    apiKey
-      ? "https://partner.semanticscholar.org/v1"
-      : "https://api.semanticscholar.org/v1"
-  );
+  const apiOrigin = "https://api.semanticscholar.org/graph/v1";
```

Comment:
Is the API key that's in place now specific to the partner site? Do we need to provision a new key for ScholarPhi to use with the new graph API?

Reply:
nope!  [Checked with Rodney and he says the same partner API key can be used for either subdomains.](https://github.com/allenai/scholar/issues/32158#issuecomment-1487527699)